### DISCUSSION TITLE
Community poll for naming this repo

### DISCUSSION THREAD

**Discussion by 2byrds** [WebOfTrust/signify-browser-extension#1] [Polls]
This open source project provides the KERI community a signify browser extension for authentication. Users will use the extension to securely leverage Signify+Keria to do simple management of AIDs and their associated keys and ACDCs — just enough to establish trust with websites.

The goal is for the extension to be available in the Chrome extension and Firefox extension catalogs.

This "authX" feature set is at the heart of what is called a "wallet" in our industry. However, in the minds of many people, "wallet" is a much bigger term. This extension intentionally has a much more limited scope, to make security audits, etc. easier.

**Comment by kentbull** [comment]
How about SigWeb?

**Reply by kentbull** [reply]
oh shoot, that's taken: https://topazsystems.com/sdks/sigweb.html

**Reply by 2byrds** [reply]
Also, I think it would be tough to distinguish what is special about "sigweb" vs. signify-ts or other signify libs that are useful for web apps.

**Comment by dhh1128** [comment]
Although I could live with a repo named "signify-browser-extension", I don't think a browser extension with that name is going to go over well. I think we need a more interesting name for the listing in the extension catalog. I could see the name "Signify" as that name, or "Polaris" or "Crux" or whatever -- but not "Signify Browser Extension".

**Reply by 2byrds** [reply]
I like the idea of a catchy name. I also think that consistency in the Signify brand is helpful (i struggle with unrelated proper nouns for significance). What about a name related to security and gaining access, something like Sentry which harkens a little to Signify? One thing we have to consider is what extension names already exist in the browser extension catalogs:
https://chromewebstore.google.com/search/Sentry
https://addons.mozilla.org/en-US/firefox/search/?q=Sentry

**Reply by 2byrds** [reply]
Polaris is getting some support and I'm good with that as well:
https://addons.mozilla.org/en-US/firefox/search/?q=Polaris&type=extension
https://chromewebstore.google.com/search/Polaris?itemTypes=EXTENSION

**Comment by kentbull** [comment]
Since the wallet is like a utility on a belt (aka Batman's belt) how about the name "Zeta", the latinized version of the largest star in Orion's belt, [Alnitak](https://en.wikipedia.org/wiki/Alnitak)?

No extension exists with that name:
- Chrome Web Store: https://chromewebstore.google.com/search/zeta
- Mozilla Extension Store: https://addons.mozilla.org/en-US/firefox/search/?q=zeta&type=extension

**Reply by 2byrds** [reply]
That is unique but it doesn't feel like it's improving our options, over other names like Polaris. I'm going to vote Polaris since it has support.

**Reply by kentbull** [reply]
It's shorter :)
Polaris is cool though, and more widely recognized.

**Reply by kentbull** [reply]
And think of this, A from ACDCs to Z in Zeta. We cover the whole alphabet. How do you get better than that?

**Comment by pfeairheller** [comment]
The name we use for the repo does not have to match the name of the published app, right?  

Leaving the name of the repo alone makes it really clear for people looking at the WoT org what this repo does.

**Reply by 2byrds** [reply]
That's fair. I generally would like to avoid multiple names for the same thing (Twitter vs. X ;p).... but if we use a external name like "Polaris" and want a more descriptive internal repo name, that can be useful.

**Reply by pfeairheller** [reply]
These aren't the same things.  One is the place where the code lives, the other is the application deployed out of that code.  Regardless, I voted and am fine living with the will of the people... all 5 of them.

**Comment by 2byrds** [comment]
Given the responses, we will keep THE REPO NAME as "signify-browser-extension" and I will start a new poll for the extension name as seen by the extension catalogs. We can revisit this repo name later with another poll after we settle on a public facing extension name.
